source/Octopus.Client/Model/BuildEnvironment.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Octopus.Client.Model
+{
+    internal enum BuildEnvironment
+    {
+        Unspecified,
```

Comment:
Worth spending half an hour to hack in other options?
Maybe use https://github.com/pvdlg/env-ci as inspiration?

Reply:
I'd definitely add in Jenkins, appveyor, bitbucket, travis at least. The more the merrier...

Reply:
yep, I'll add the extras.